### PR TITLE
[FEAT] expose current metric values

### PR DESCRIPTION
**Pull request by bmiguel-teixeira**
**What this PR does / why we need it**:
This PR exports the current values of the metrics. We already have the spec target values exported. Together with this PR, the user will be able to have complete information on current vs target more easily.

**How does this change affect the cardinality of KSM**: *(increases, decreases or does not change cardinality)*
Increase metric count, cardinality remains equal to spec_target.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: internal/store/horizontalpodautoscaler.go [hunk 1]
```diff
@@ -184,6 +184,68 @@ func hpaMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generat
 				return &metric.Family{Metrics: ms}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_horizontalpodautoscaler_status_current_metric",
+			"The current value of the metric defined in the spec section.",
```
Comment (fpetkovski) [inline internal/store/horizontalpodautoscaler.go:189]:
```suggestion
			"The current value of the metric defined in the spec.",
```

**Review by fpetkovski — COMMENTED** [review]
Looks good, one small nit.

**Comment by fpetkovski** [comment]
We also need the CI to pass

File: internal/store/horizontalpodautoscaler.go (hunk 1 above)
Comment (bmiguel-teixeira) [reply internal/store/horizontalpodautoscaler.go]: no problem. updated.
Also fixed CI issue, did not realize that `make container` did not run them.
ty

**Comment by mrueg** [comment]
/lgtm

/hold
for @fpetkovski to cancel

**Comment by mrueg** [comment]
@bmiguel-teixeira Friendly ping on my message above. Could you clarify the authorship of the code submitted in this PR? Otherwise we would need to remove the commits by the end of the week.

**Comment by mrueg** [comment]
As there was no response from the author within the last 15 days and the source remains not clear, I removed the commits from our master branch (force pushed) as agreed with the other maintainers.

**Comment by brunomiguel-teixeira** [comment]
Hi @mrueg
 I am afraid I did not see the comments. I was unaware that there was a previously opened PR adding a similar functionality. While the PR I opened is fully from my ownership and built from our own use-cases, as long as the same functionality is made available its fine.

**Comment by bmiguel-teixeira** [comment]
Hi @mrueg
I am afraid I did not see the comments. I was unaware that there was a previously opened PR adding a similar functionality. While the PR I opened is fully from my ownership and built from our own use-cases, as long as the same functionality is made available its fine.

**Pull request merged**